### DISCUSSION TITLE
Train with the PGN I imported

### DISCUSSION THREAD

**Discussion by 44ayoub** [franciscoBSalgueiro/en-croissant#475] [Q&A]
First of all thank you for the nice open-source software. I imported a PGN-File and tried to use the puzzles that are in the file. Is it possible to use the Puzzle feature for local databases? Because I only see the Lichess option and it would be nice to train with my own file. Thanks for the help

**Comment by faurehu** [comment]
If I understand you correctly, it seems like you are looking for the repertoire feature. Here are the instructions: https://encroissant.org/docs/guides/manage-repertoire
